pxc/local/test-binary-pxc

```diff
@@ -96,17 +65,137 @@ if [[ "${DEFAULT_TESTING}" != "no" ]]; then
     else
         MYSQLD_ENV="${ADD_TO_LD_PRELOAD:-}${JEMALLOC}:${EATMYDATA}"
     fi
-
-  MTR_BUILD_THREAD=auto ./mysql-test-run.pl \
-        --parallel=${PARALLEL} \
-        --result-file \
-        ${MTR_ARGS} \
-        --force \
-        --max-test-fail=0 \
-        --suite-timeout=9999 \
-        --testcase-timeout=${TESTCASE_TIMEOUT} \
-        | tee ${WORKDIR_ABS}/mtr.output || status=$?
-    process_mtr_output PXC "${WORKDIR_ABS}/mtr.output" "${WORKDIR_ABS}/junit.xml"
+    if [ ${FULL_MTR} == "no" ]; then
+        # galera suite
+        if echo "$MTR_SUITES" | grep -Ewq "galera"; then
```

Comment:
This implementation will ignore any non default value (innodb, gr, x, main, ...) specified to the MTR_SUITES argument, which is a bad idea: as jenkins only report failures, this implies that the entire suite succeeded, while in truth it wasn't even ran.

Please make sure that anything specified by the user in these configuration values will be correctly used by the script.

Reply:
Updated MTR script to support non-default MTR suites. Custom configurations will be used correctly through MTR_ARGS.